docs/.vitepress/config.js

```diff
@@ -126,6 +145,262 @@ export default defineConfig({
             { text: 'Designers guide', link: '/migration/designers/' },
           ],
         },
+        {
+          text: 'Classes',
```

Comment:
Should this perhaps be named "CSS" or something instead? 🤔 
![image](https://github.com/warp-ds/tech-docs/assets/7531505/2327bcaf-a147-4415-b29f-deaae11ef074)